configs/components/nemo/nemo.yaml

```diff
@@ -609,18 +648,41 @@ namelist_changes:
                         nn_rstctl: ${nn_rstctl} 
                         ln_rstart: ${nemo.lresume}
                 namtsd:                                                
-                        ln_tsd_tradmp: ${ln_tsd_tradmp} 
+                        #ln_tsd_tradmp: ${ln_tsd_tradmp} 
                         ln_tsd_init: ${ln_tsd_init} 
-                namdom:
-                        nn_closea: 1
-                        nn_msh: ${nn_msh} 
-                        rn_rdt: ${time_step}
+                #namdom:
```

Comment:
I would always advocate for deleting code rather than just commenting it. If needed we can always go into the git history. But, that's just my personal style.

Reply:
You are right. I'll push another commit